### PR TITLE
Improve ginkgo installation script

### PR DESCRIPTION
**Pull request by bnshr**
This is an improvement on Ginkgo installation.
We want to install a specific version that has been used in the TNF repo.

### CODE REVIEW

**Review by theemj — APPROVED** [review]
looks good and works on my test cluster :+1:

File: scripts/install-ginkgo.sh [hunk 1]
```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env bash
 
-GOPATH="/root/go"
+REQUIRED_GINKGO_VERSION="2.9.2"
```
Comment (sebrandon1) [inline scripts/install-ginkgo.sh:3]: Required version should get brought in from the go.mod:

`$(awk '/ginkgo/ {printf "%s/ginkgo@%s", $1, $2}' go.mod)`

**Review by sebrandon1 — CHANGES_REQUESTED** [review]
Make the required version the one specified in the go.mod.

File: scripts/install-ginkgo.sh [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
-	go install "$(awk '/ginkgo/ {printf "%s/ginkgo@%s", $1, $2}' go.mod)"
-} fi
+function version { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+if [ "$(version "${INSTALLED_GINKGO_VERSION}")" -ge "$(version "${REQUIRED_GINKGO_VERSION}")" ]; then
+    echo "Version is up to date"
+else
+        GINKGO_TMP_DIR=$(mktemp -d)
+        cd "$GINKGO_TMP_DIR" || exit
+        go mod init tmp
```
[the remark this answers is not in the document]
Comment (bnshr) [reply scripts/install-ginkgo.sh:16]: Yeah, that's right.

**Review by sebrandon1 — APPROVED** [review]

**Comment by bnshr** [comment]
Thanks!

**Pull request merged**